### ISSUE TITLE
CRASH

### ISSUE THREAD

**Issue by FunnyCraft** [PaperMC/Paper#337]
http://pastebin.com/1Kt4DcNq

**Comment by Cat7373** [comment]
Why are there so many `Async Chat Thread`?

**Comment by Cat7373** [comment]
```
[10:27:06] [Paper Watchdog Thread/ERROR]: Current Thread: Server thread
[10:27:06] [Paper Watchdog Thread/ERROR]:   PID: 16 | Suspended: false | Native: false | State: RUNNABLE
[10:27:06] [Paper Watchdog Thread/ERROR]:   Thread is waiting on monitor(s):
[10:27:06] [Paper Watchdog Thread/ERROR]:       Locked on:net.minecraft.server.v1_10_R1.ServerConnection.c(ServerConnection.java:152)
```

**Comment by Cat7373** [comment]
Please provide configuration file: bukkit.yml, spigot.yml, paper.yml, server.properties
and plugin list

**Comment by FunnyCraft** [comment]
hold

https://www.sendspace.com/file/hfljka

**Comment by zachbr** [comment]
This is pretty old, if you're still seeing this, open a new ticket and provide more recent crash logs.
Thanks.
